presto-main/src/test/java/io/prestosql/sql/query/TestExpressions.java

```diff
@@ -34,21 +34,6 @@ public void teardown()
         assertions = null;
     }
 
-    @Test
-    public void testBooleanExpressionInCase()
-    {
-        assertions.assertQuery("VALUES CASE 1 IS NULL WHEN true THEN 10 ELSE 20 END", "VALUES 20");
-        assertions.assertQuery("VALUES CASE 1 IS NOT NULL WHEN true THEN 10 ELSE 20 END", "VALUES 10");
-        assertions.assertQuery("VALUES CASE 1 BETWEEN 0 AND 2 WHEN true THEN 10 ELSE 20 END", "VALUES 10");
-        assertions.assertQuery("VALUES CASE 1 NOT BETWEEN 0 AND 2 WHEN true THEN 10 ELSE 20 END", "VALUES 20");
-        assertions.assertQuery("VALUES CASE 1 IN (1, 2) WHEN true THEN 10 ELSE 20 END", "VALUES 10");
-        assertions.assertQuery("VALUES CASE 1 NOT IN (1, 2) WHEN true THEN 10 ELSE 20 END", "VALUES 20");
-        assertions.assertQuery("VALUES CASE 1 = 1 WHEN true THEN 10 ELSE 20 END", "VALUES 10");
-        assertions.assertQuery("VALUES CASE 1 = 2 WHEN true THEN 10 ELSE 20 END", "VALUES 20");
-        assertions.assertQuery("VALUES CASE 1 < 2 WHEN true THEN 10 ELSE 20 END", "VALUES 10");
-        assertions.assertQuery("VALUES CASE 1 > 2 WHEN true THEN 10 ELSE 20 END", "VALUES 20");
-    }
-
```

Comment:
Is there any reason? It looks like there is no change in the original one.
https://github.com/trinodb/trino/blob/4bcd5b5c35cddd3c076a616282aaf57168236052/core/trino-main/src/test/java/io/trino/sql/query/TestExpressions.java#L40

Reply:
This is the same, this test fails as we haven't cherry-picked related changes

Reply:
Alright